docker/centos.dockerfile

```diff
@@ -0,0 +1,30 @@
+# vim: set ft=dockerfile:
+
+FROM usgsastro/centos:latest
+
+SHELL ["/bin/bash", "-lc"]
```

Comment:
Are these docker containers set up to build on push? And if they rely on a docker container we already maintain, why not put it all here?

Reply:
Not set up to build on push yet, I figured I'd wait to get feedback from you guys. I included them here so that anyone wanting to build isis could easily use them after cloning the repo. 

As for the FROM block, the image it inherits from is defined https://github.com/USGS-Astrogeology/docker. The repo includes base images for each OS, as well as a whole bunch of others that include the SSL root cert for internal use. Which is why I didn't include the parent image here.